src/sqltest/workbench/api/extHostAccountManagement.test.ts

```diff
@@ -366,17 +366,9 @@ suite('ExtHostAccountManagement', () => {
 		extHost.$getAllAccounts()
 			.then((accounts) => {
```

Comment:
@MattIrv these are the changes I made. Please take a look. Basically this test never worked as expected, at least since last changed by Vincent.